src/inversion_scripts/merge_partial_k.py

```diff
@@ -0,0 +1,94 @@
+import os
+import sys
+import yaml
+import pickle as pickle
+import numpy as np
+import xarray as xr
+from src.inversion_scripts.utils import load_obj, calculate_superobservation_error
+
+
+def merge_partial_k(satdat_dir, lat_bounds, lon_bounds, obs_err, background=False):
```

Comment:
This function would probably benefit from a long-form description / comment block.